pkg/local_object_storage/metabase/metadata.go

```diff
@@ -794,3 +807,10 @@ func (x *metaAttributeSeeker) restoreVal(id []byte, attr string, stored []byte)
 	}
 	return string(stored), nil
 }
+
+func convertFilterValue(f object.SearchFilter) (object.SearchMatchType, string) {
+	if attr := f.Header(); attr == object.FilterRoot || attr == object.FilterPhysical {
+		return object.MatchStringEqual, binPropMarker
```

Comment:
Do we have a check anywhere for these restricted (per attribute) operations?

Reply:
no checks. U mean that ROOT and PHY aint used w/ other matchers?

Reply:
Yes, it should be a part of the request semantics check.

Reply:
gonna do in #3111